src/types/Filters.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filters {
```

Comment:
it's better to use `const enum`. There is less code overhead.

See the difference here https://phuoc.ng/collection/this-vs-that/const-enum-vs-enum/

Reply:
Can't use const enum in this case.